docs/overview/diagrams.md

```diff
@@ -236,3 +236,69 @@ classDiagram
+
+
+# AssociativeBoth
+
+[![AssociativeBoth hierarchy][AssociativeBoth-image]][AssociativeBoth-link]
+
+<details><summary>Mermaid</summary>
+
+```mermaid
+classDiagram
+  AssociativeBoth~F<_>~ <|-- CommutativeBoth~F<_>~
+  AssociativeBoth~F<_>~ <|-- IdentityBoth~F<_>~
+  class AssociativeBoth~F<_>~{
+    Chunk
+    Exit
+    Fiber
+    NonEmptyChunk
+    AndF[Schedule]
+    OrF[Schedule]
+    Vector
+    ZIO
```

Comment:
Also, for type which have some type parameters as free arguments, could you please write it like this `ZIO[R, E, +*]`?

On the other hand, types which have all type parameters bound, like `Vector` are fine like this, no need to write it as `Vector[+*]`.

Reply:
I saw the change made by Ernesto and sth like Vector[+*] I recognized as a convention.

Reply:
I'm terribly sorry for the confusing messages and going back and forth on this. We're still settling on the conventions.
After some thinking, it seems to me `Vector[+*]` is preferred (because you see if it's co- or contra-variant) and it's also more regular.